e2e/scenario_test/settings/not_set/basal_rates/settings.no_basal_rates.spec.js

```diff
@@ -1,47 +1,24 @@
-const { setup, pump, cgm, match, settings } = require('../../../../../src/index');
+const { setup, match, settings, loopSettings } = require('../../../../../src/index');
```

Comment:
[nit] Perhaps one thing that could help the "../../.." issue would be to get rid of the deep directory hierarchy and focus on a standard naming scheme for the tests. So, perhaps this file would be "e2e/scenario_test/settings.not_set.basal_rates.spec.js". Also, consider that each of these test files may be common to one or more "groups" of tests (scenario vs. smoke).